lib/bitcoin/interfaces/IBitcoinWallet.ts

```diff
@@ -0,0 +1,36 @@
+import { Address, Script, Transaction } from 'bitcore-lib';
+
+/**
+ * Represents a bitcoin wallet.
+ */
+export default interface IBitcoinWallet {
+
+  /**
+   * Gets the public key associated with this wallet.
+   *
+   * @returns The public key associated with this wallet as Buffer.
+   */
+  getPublicKey (): Buffer;

```

Comment:
Consider 










c



```suggestion
  getPublicKeyHex (): string;

```
Buffer is too generic without further specification on what the serialization format is, consider just use hex directly since it looks like that's what is used later on.

Reply:
Good idea.